apps/wpcom-block-editor/src/wpcom/features/live-preview/hooks/use-previewing-theme.ts

```diff
@@ -19,6 +20,25 @@ const getThemeType = ( theme?: Theme ) => {
 	if ( isPremiumTheme ) {
 		return PREMIUM_THEME;
 	}
+
+	// @TODO Replace all the logic above with the following code once Theme Tiers is live.
+	if ( config.isEnabled( 'themes/tiers' ) ) {
+		return theme?.theme_tier?.slug ?? undefined;
+	}
+
+	return undefined;
+};
+
+/**
+ * Get the theme required feature.
+ * This only support WooCommerce and Premium themes.
```

Comment:
this isn't true

Reply:
Why you little nitpicker! Fixed, though, we're going to have to generalize or remove this description once we clean up the non-tier-related code.